govcd/vm.go

```diff
@@ -459,10 +462,10 @@ func (vm *VM) Customize(computername, script string, changeSid bool) (Task, erro
 		HREF:                vm.VM.HREF,
 		Type:                types.MimeGuestCustomizationSection,
 		Info:                "Specifies Guest OS Customization Settings",
-		Enabled:             true,
+		Enabled:             takeBoolPointer(true),
```

Comment:
Maybe it is worth it also change this harcoded value too?

Reply:
This function does not have a parameter for this field, while `changeSid` had, but was missed.